clients/src/main/resources/common/message/FetchRequest.json

```diff
@@ -53,7 +53,9 @@
   //
   // Version 15 adds the ReplicaState which includes new field ReplicaEpoch and the ReplicaId. Also,
   // deprecate the old ReplicaId field and set its default value to -1. (KIP-903)
-  "validVersions": "0-15",
+  //
+  // Version 16 is the same as version 15.
+  "validVersions": "0-16",
```

Comment:
we should mark these APIs as in development by specifying `"latestVersionUnstable":true`
See https://github.com/apache/kafka/pull/14046 for an example.

Reply:
I'm not sure about this. The intention is for all the new fields to be optional, the client should be able to handle them not being populated (by doing a full metadata refresh), so it doesn't seem unstable to me. For reference FetchResponse already has the CurrentLeader field we're adding to ProduceResponse, we just never used it for anything.

Reply:
It is unstable since we didn't implement it yet :) 
We can remove that once we implement.

Reply:
ok, added the label